README.md

```
@@ -1,72 +1,239 @@
-1. Выполните шаги 1 и 2 из инструкции "Как запустить приложение"
+## 🛠️ Сборка исполняемого файла (.EXE из исходников)
 
-2. Выполните комманду `make build` или `make`
+Если вы внесли изменения в код и хотите собрать собственный `.exe` файл, воспользуйтесь настроенным `Makefile`:
 
-3. Запустите исполняемый файл из директории dist
+```bash
+make build
+```
+*Собранное приложение со всеми ресурсами и конфигурациями логгера будет сохранено в папку `dist/`.*
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)